content/livesync/index.textile

```diff
@@ -1,6 +1,45 @@
 ---
 title: What is LiveSync?
+product: livesync
 ---
 
+Livesync keeps your frontend clients and your database consistently in sync. It ensures a smooth  and fast user experience through optimistic updates. LiveSync is built on top of Ably’s global message delivery network, guaranteeing reliable messaging delivery and low latency. 
 
+LiveSync is made up of two key components:
 
+* *Database Connector.* A backend service for broadcasting changes from your database in realtime.
+
+* *Models SDK.* A frontend library that simplifies subscribing to changes in data models and applying optimistic updates, merging them with confirmed updates.
+
+<a href="/images/diagrams/what-is-livesync.png" target="_blank">
+  <img src="/images/diagrams/what-is-livesync.png" style="width: 100%" alt="Livesync diagram">
```

Comment:
Is this image checked in to this repo?

Reply:
Just figuring out why this hasn't been pushed -- It's only a placeholder diagram for now FYI

Reply:
Change made here: [ee5a18f](https://github.com/ably/docs/pull/2054/commits/ee5a18f2d85a916eb0eea7262368d22962e64f38)